### PR TITLE
Add bathymetry approximation function

### PR DESCRIPTION
**Pull request by jsta**
See #80

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    testthat
-License: GPL (>= 2)
-Packaged: 2014-07-06 09:09:24 UTC; Luke
-Repository: https://github.com/GLEON/rLakeAnalyzer
-Date/Publication: 2015-04-02 12:00:00
-RoxygenNote: 5.0.1
+Package: rLakeAnalyzer
+Title: Lake Physics Tools
+Maintainer: Luke Winslow <lawinslow@gmail.com>
+Version: 1.8.8
```
Comment (lawinslow) [inline DESCRIPTION]: Could you increment the version? I'd say a minor version jump for the addition of a function. 1.9.0

File: DESCRIPTION [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+    number, Schmidt stability and others.
+Imports:
+    plyr
+Suggests:
+    testthat
+License: GPL (>= 2)
+Packaged: 2014-07-06 09:09:24 UTC; Luke
+Repository: https://github.com/GLEON/rLakeAnalyzer
+Date/Publication: 2015-04-02 12:00:00
+RoxygenNote: 6.0.1
```
Comment (lawinslow) [inline DESCRIPTION]: Also, line endings changed in this file. Could you switch so we get a proper diff?

File: R/helper.functions.R [hunk 3]
```diff
@@ -1,133 +1,153 @@
...
+#'
+#' @examples \dontrun{
+#' approx.hypso(zMax = 25, lkeArea = 39400000, numZ = 15)
+#' }
+approx.hypso <- function(zMax, lkeArea, numZ){ 
+  depth	<- seq(0, zMax, length.out = numZ)
+  area  <- approx(c(0, zMax), c(lkeArea, 0), depth)$y
+  
+  data.frame(depths = depth, areas = area)
+}
```
Comment (lawinslow) [inline R/helper.functions.R]: Line ending fix please.

File: man/thermo.depth.Rd [hunk 4]
```diff
@@ -1,57 +1,56 @@
...
+ cat('The thermocline depth is:', t.d)
+
+}
+\seealso{
+\code{\link{ts.thermo.depth}}, \code{water.density}
+}
+\author{
+Luke Winslow
+}
+\keyword{manip}
```
Comment (lawinslow) [inline man/thermo.depth.Rd]: Line endings...

File: man/wtr.plot.temp.Rd [hunk 5]
```diff
@@ -1,38 +1,37 @@
...
+}
+
+}
+\seealso{
+\code{\link{load.ts}} and \code{\link{wtr.lineseries}}
+}
+\author{
+Jennifer Brentrup, Taylor Leach, Luke Winslow
+}
+\keyword{hplot}
```
Comment (lawinslow) [inline man/wtr.plot.temp.Rd]: line endings...

File: man/meta.depths.Rd [hunk 6]
```diff
@@ -1,58 +1,57 @@
...
+\references{
+Wetzel, R. G. 2001. Limnology: Lake and River Ecosystems, 3rd ed. Academic Press.
+}
+\seealso{
+\code{\link{ts.meta.depths}}, \code{\link{thermo.depth}}
+}
+\author{
+Jennifer Brentrup, Luke Winslow
+}
+\keyword{manip}
```
Comment (lawinslow) [inline man/meta.depths.Rd]: line endings.

File: R/helper.functions.R [hunk 7]
```diff
@@ -1,133 +1,153 @@
...
+#' @param zMax numeric max depth in meters
+#' @param numZ integer number of depth layers
+#' @param lkeArea numeric lake area in meters squared.
+#'
+#' @export
+#'
+#' @examples \dontrun{
+#' approx.hypso(zMax = 25, lkeArea = 39400000, numZ = 15)
+#' }
+approx.hypso <- function(zMax, lkeArea, numZ){
```
Comment (lawinslow) [inline R/helper.functions.R]: I know there's some certainly some debate here, but due to past usage, it is generally referred to as `bathy`. Could we go with that naming for consistency?

File: DESCRIPTION (hunk 1 above)
Comment (jsta) [inline DESCRIPTION]: Hmm, all the line endings are all automated with `devtools::document()`. I could revert the roxygen update...

File: DESCRIPTION (hunk 2 above)
Comment (jsta) [inline DESCRIPTION]: Hmm, all the line endings are all automated with `devtools::document()`. I could revert the roxygen update...

File: DESCRIPTION (hunk 1 above)
Comment (lawinslow) [inline DESCRIPTION]: Or you can change your git settings. Or you could just edit the files to fix. Either way, I prefer being able to trace line changes back when possible.

Comment (jsta) [inline DESCRIPTION]: I'm afraid I don't understand. I think the best course of action here might be for me to cancel the PR and resubmit only adding the function.

Comment (lawinslow) [inline DESCRIPTION]: https://stackoverflow.com/a/11341700/1596891

You can just fix the file line endings and commit the changes.

Comment (jsta) [inline DESCRIPTION]: Setting line endings in `.gitattributes` didn't seem to make any difference.

**Comment by jsta** [comment]
I used the documentation at https://help.github.com/articles/dealing-with-line-endings/. Happy to cancel and resubmit a minimal PR if this is incorrect.

**Comment by lawinslow** [comment]
Still problematic. Please fix or resubmit a new PR.

**Pull request closed without merging**